generators/kubernetes/templates/_deployment.yml

```diff
@@ -61,5 +61,11 @@ spec:
         - name: SPRING_DATA_ELASTICSEARCH_CLUSTER_NODES
           value: <%= app.baseName.toLowerCase() %>-elasticsearch.<%= kubernetesNamespace %>.svc.cluster.local:9300
         <%_ } _%>
+        <%_ if (app.messageBroker === 'kafka') { _%>
+        - name: SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS
+          value: <%= app.baseName.toLowerCase() %>-kafka
```

Comment:
Yes, I had set this to the shorter name because it was not working on my minikube at that time. I'll change it.